specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/GA/2021-02-01/examples/GetClusterExample.json

```diff
@@ -0,0 +1,30 @@
+{
+  "parameters": {
+    "subscriptionId": "1bfbb5d0-917e-4346-9026-1d3b344417f5",
+    "resourceGroupName": "k8sc-rg",
+    "clusterName": "testCluster",
+    "api-version": "2021-02-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "72f988bf-86f1-41af-91ab-2d7cd011db47",
```

Comment:
Here as well we would see prov state right ?.

Reply:
Same comment for other responses below as well.

Reply:
Updated old examples as well